integreat_cms/release_notes/current/unreleased/2076.yml

```
@@ -0,0 +1,2 @@
+en: A background job has been introduced that deletes unactivated accounts after 7 days.
+de: Es wurde ein Hintergrundjob eingeführt, der nicht aktivierte Accounts nach 7 Tagen löscht.
```

Nit: We had agreed in the past to write release notes in imperative form. So in this case it would be something like

```suggestion
de: Führe Hintergrundjob ein, der nicht aktivierte Accounts nach 7 Tagen löscht
```

---

Thank you @JoeyStk :)